pymatgen/command_line/bader_caller.py

```diff
@@ -98,19 +96,18 @@ class BaderAnalysis:
               "BaderAnalysis requires the executable bader to be in the path."
               " Please download the library at http://theory.cm.utexas"
               ".edu/vasp/bader/ and compile the executable.")
-    def __init__(self, chgcar_filename, potcar_filename=None,
-                 chgref_filename=None, parse_atomic_densities=False):
+    def __init__(self,
+                 cube_filename=None,
+                 chgcar_filename=None,
+                 potcar_filename=None,
+                 chgref_filename=None,
+                 parse_atomic_densities=False):
         """
         Initializes the Bader caller.
 
         Args:
```

Comment:
It looks like this docstring is incomplete

Reply:
Should be fixed now.

Reply:
Thanks!